setup.py

```
@@ -368,13 +375,21 @@ def _pythonpath_env(extra_path):
+        ),
     ]
+    + ([eeasisss_ext] if build_eeasisss else [])
 )
 
 print("BUILD_BACKEND: {}".format(BUILD_BACKEND))
 
 README = "README.md"
+LONG_DESCRIPTION = ""
+
+try:
+    with open(README, mode="r", encoding="utf-8") as readme_file_ptr:
```

Ruff (0.14.10)

Unnecessary mode argument

Remove mode argument

(UP015)